torchci/rockset/pytorch_dev_infra_kpis/__sql/time_to_signal.sql

```diff
@@ -37,8 +61,8 @@ select
 from
     tts_by_sha
 where
-    job_cnt > 15
+    job_cnt > 35
```

Comment:
Parameterize this?

Reply:
That's a crude filter to remove workflows that aren't pull or trunk. This really needs to be replaced with a better filter that more accurately removes other workflows (like periodic, nightly, etc), but that's a fix for later.

Making this a parameter would just introduce more changes required when this query is improved.